notebooks/examples/CEMS.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
```

Comment:
<p>Man, is Texas really that bad? This seems like a crazy outlier. Might there be some bad data somewhere? It might be more illustrative to plot the GHG emissions per capita on a state by state basis, since you've got the census data handy already.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>